sections/email-signup-banner.liquid

```diff
@@ -342,7 +342,7 @@
         {
           "type": "richtext",
           "id": "text",
-          "default": "<p>Be the first to know when we launch.</p>",
+          "default": "t:sections.email-signup-banner.blocks.paragraph.settings.paragraph.default",
```

Comment:
Ah yeah, related to my previous comment, we should keep the `<p>` tags here in the liquid file, rather than the translations.